### PR TITLE
Added title, desc & link to homepage

### PR DESCRIPTION
**Pull request by stephlaf**
@reemgawad Please review & merge if all good.

What changed:

- Added a title
- Added a description
- Added a link to nowhere for now

How to test:
GOTO `/`

Screen Shot 2022-08-15 at 09 59 10

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,6 +1,3 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
-
-<% if user_signed_in? %>
-  <h2>Welcome <%= current_user.email %></h2>
-<% end %>
+<h2>This is our Homepage</h2>
```
Comment (reemgawad) [inline app/views/pages/home.html.erb:1]: This should be an <h1> instead.

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,6 +1,3 @@
...
+<p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Iure animi veritatis cupiditate voluptate ipsum, unde consectetur, minus, repudiandae officia obcaecati sequi repellat? Delectus nam id porro, nulla veritatis quos est?</p>
+<a href="#">Click me!</a>
```
Comment (reemgawad) [inline app/views/pages/home.html.erb:3]: Note: We should figure out where this link goes

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (stephlaf) [reply app/views/pages/home.html.erb:1]: Aw man, you're right. On it now!

File: app/views/pages/home.html.erb (hunk 2 above)
Comment (stephlaf) [reply app/views/pages/home.html.erb:3]: That will be for the next sprint.

**Pull request merged**